examples/with-imagekit/utils/types.ts

```diff
@@ -0,0 +1,8 @@
+import { FileObject } from "imagekit/dist/libs/interfaces";
```

Comment:
Is there no way to directly import it w/o referring to `dist/libs/interfaces`?

Reply:
As discussed on a call, we will keep it like this for now until we expose interfaces in node.js SDK